frontend/www/js/omegaup/arena/rankingStore.ts

```diff
@@ -0,0 +1,50 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+import { types } from '../api_types';
+import { omegaup } from '../omegaup';
+
+Vue.use(Vuex);
+
+export interface RankingState {
+  miniRankingUsers: omegaup.UserRank[];
+
+  ranking: types.ScoreboardRankingEntry[];
+
+  rankingChartOptions: Highcharts.Options;
+
+  lastTimeUpdated: null | Date;
+}
+
+export const rankingStoreConfig = {
+  state: {
+    miniRankingUsers: [],
+    ranking: [],
+    rankingChartOptions: {},
+    lastTimeUpdated: null,
+  },
+  mutations: {
+    updateMinirankingUsers(
```

Comment:
```suggestion
    updateMiniRankingUsers(
```

sorry, se me fue esto >< puede llamarse `MiniRanking` por consistencia?

Reply:
Listo